app-policy/config/cluster/node-config.yaml

```diff
@@ -57,8 +57,6 @@ coreos:
         --cluster-dns=10.100.0.10 \
         --cluster-domain=cluster.local \
         --hostname-override=$private_ipv4 \
-        --logtostderr=true \
-        --network-plugin=cni \
```

Comment:
Was the removal of `--network-plugin=cni` here intentional? Has it also been deprecated?

Reply:
Yes it was intentional, because of this commit: https://github.com/projectcalico/calico/commit/25a614480b3c48a1feadf46daaff6c41bf1705f1 (though I admit I didn't research on which versions exactly this one was deprecated/removed)